### DISCUSSION TITLE
New module that counts different characters

### DISCUSSION THREAD

**Discussion by gmetais** [macbre/phantomas#907] [Ideas]
Hi there, I'm starting the first discussion :)

I used to have a feature in YLT that compares the characters inside a font with all the characters on the page (the idea is to detect if some Unicode subsets could be removed from the font). It was using a custom module I wrote for phantomas v1. Now with v2, it is not possible to add custom modules anymore.

The module: https://github.com/gmetais/phantomas/tree/charactersCount/modules/charactersCount

What's your favorite way to work? Should I create a pull-request and add this module into phantomas (even if it not really interesting for other phantomas users)? Or should we re-introduce custom modules?

**Comment by macbre** [comment]
Thanks for starting the discussion here 🙂 

Let's reintroduce custom modules.
